### ISSUE TITLE
Supervoxel clustering tutorial does not compile

### ISSUE THREAD

**Issue by sotte** [PointCloudLibrary/pcl#1177]
The [supervoxel clustering tutorial](http://pointclouds.org/documentation/tutorials/supervoxel_clustering.php) does not compile.

Steps to reproduce:
```
mkdir supervoxel_clustering
cd supervoxel_clustering
curl -O http://pointclouds.org/documentation/tutorials/_downloads/supervoxel_clustering.cpp
curl -o CMakeLists.txt http://pointclouds.org/documentation/tutorials/_downloads/CMakeLists2.txt
curl -o milk_cartoon_all_small_clorox.pcd https://github.com/PointCloudLibrary/data/blob/master/tutorials/correspondence_grouping/milk_cartoon_all_small_clorox.pcd?raw=true -o milk_cartoon_all_small_clorox.pcd

mkdir build
cd build
cmake ..
make
```

Error message
```
[100%] Building CXX object CMakeFiles/supervoxel_clustering.dir/supervoxel_clustering.cpp.o
/home/stefan/git/pcl_workspace/supervoxel_clustering/supervoxel_clustering.cpp: In function 'void addSupervoxelConnectionsToViewer(PointT&, PointCloudT&, std::string, boost::shared_ptr<pcl::visualization::PCLVisualizer>&)':
/home/stefan/git/pcl_workspace/supervoxel_clustering/supervoxel_clustering.cpp:158:11: error: invalid use of incomplete type 'class vtkPolyLine'
   polyLine->GetPointIds  ()->SetNumberOfIds(points->GetNumberOfPoints ());
           ^
In file included from /usr/include/vtk-5.8/vtkPolyDataAlgorithm.h:38:0,
                 from /usr/include/vtk-5.8/vtkVectorText.h:35,
                 from /usr/include/pcl-1.7/pcl/visualization/impl/pcl_visualizer.hpp:44,
                 from /usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:2004,
                 from /home/stefan/git/pcl_workspace/supervoxel_clustering/supervoxel_clustering.cpp:5:
```

I'm using Ubuntu 14.04 with pcl 1.7.1.

**Comment by taketwo** [comment]
Try to add the following include:

```cpp
#include <vtkPolyLine.h>
```

**Comment by jpapon** [comment]
Should I add this include to pull request #1115?

**Comment by taketwo** [comment]
Yep, makes sense.

**Comment by sotte** [comment]
Thanks for the quick reply! I'll leave this open till #1115 is merged.

**Comment by taketwo** [comment]
Fixed via #1115.
